package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "hathor-wallet-headless",
+  "version": "1.0.0",
+  "description": "Hathor Wallet Headless, i.e., without graphical user interface",
+  "main": "index.js",
+  "dependencies": {
+    "@babel/cli": "^7.8.3",
+    "@babel/core": "^7.8.3",
+    "@babel/node": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@hathor/wallet-lib": "file:../hathor-wallet-lib",
```

Comment:
We should add the version of the package here

Reply:
Exactly. I'm just waiting for the release of the new version.

Reply:
`0.9.0` released

Reply:
Fixed. Thanks!